server/src/main/resources/application.yml

```diff
@@ -44,8 +43,12 @@ micronaut:
           - isAuthenticated()
     reject-not-found: false
     redirect:
+      login-success: /
+      on-rejection: /oauth/login/google
       unauthorized:
         url: /oauth/login/google
+      forbidden:
+        url: /forbidden
     session:
```

Comment:
We should remove lines 51-53 since it is deprecated.

https://micronaut-projects.github.io/micronaut-security/snapshot/guide/configurationreference.html#io.micronaut.security.token.config.TokenConfigurationProperties (end-of-file)

Reply:
Done